Specs/NumberWithUnit/Swedish/CurrencyModel.json

```diff
@@ -0,0 +1,425 @@
+    "Results": [
+      {
+        "Text": "100元",
+        "TypeName": "currency",
+        "Resolution": {
+          "unit": "Chinese yuan",
+          "value": "100",
+          "isoCurrency": "CNY"
+        }
+      }
+    ]
+
+  },
+  {
+    "Input": "Svenske pizzamiljardären säljer aktier för 70 miljoner kronor",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "70 miljoner kronor",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "70000000",
+          "unit": "Krone"
+        },
+        "Start": 43,
+        "End": 60
+      }
+    ]
+  },
+  {
+    "Input": "det finska företaget köpte tillgångar till ett värde av 420 miljoner finska mark .",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "420 miljoner finska mark",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "420000000",
+          "unit": "Finnish markka"
+        },
+        "Start": 56,
+        "End": 79
+      }
+    ]
+  },
+  {
+    "Input": "DNB Markets sänker riktkursen för Ericsson till 134 kronor från 138 kronor. Rekommendationen köp upprepas. Det framgår av en analys. Aktien byter ägare på 102,60 kronor i skrivande stund på torsdagsförmiddagen.",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "134 kronor",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "134",
+          "unit": "Krone"
+        },
+        "Start": 48,
+        "End": 57
+      },
+      {
+        "Text": "138 kronor",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "138",
+          "unit": "Krone"
+        },
+        "Start": 64,
+        "End": 73
+      },
+      {
+        "Text": "102,60 kronor",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "102,6",
+          "unit": "Krone"
+        },
+        "Start": 155,
+        "End": 167
+      }
+    ]
+  },
+  {
+    "Input": "Bara under måndagens rally i de svaga aktierna tappade blankare på de sju reseaktierna Carnival Corp, Expedia Group, Booking Holdings Inc , Royal Caribbean Group, American Airlines Group Wynn Resorts och Norwegian Cruise Line Holdings, 2,35 miljarder dollar.",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "2,35 miljarder dollar",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "2350000000",
+          "unit": "Dollar"
+        },
+        "Start": 236,
+        "End": 256
+      }
+    ]
+  },
+  {
+    "Input": "priset för den andra simulatorn varierar med 16,4 miljoner US$",
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "16,4 miljoner us$",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "16400000",
+          "unit": "United States dollar"
```

Comment:
There should be a isoCurrency property here. How does the test pass without it?
Can you double check if some validation is not happening properly?

Reply:
I found out that I called the TestNumberWithUnit test instead of the TestCurrency test for the Currency model. I also updated the extractor to follow the English one. Now it all works. Thanks for making me aware of this :)